dist/profile/templates/jenkinscontroller/casc/clouds.yaml.erb

```diff
@@ -479,6 +479,8 @@ jenkins:
           #!/bin/sh
           set -eux
 
+          systemctl stop sshd.service
```

Comment:
why?

Reply:
by locking the sshd at the beginning and restarting it at the end, we ensure that the agent isn't seen as ready as long as the cloud-init is not done ... (it's already assumed by the loop within the initscript but this loop isn't working on windows) as we set the new mount point as the workspace, we need the agent to wait until the mount is done, not to write in the wrong spot